source/sass/patterns/molecules/inline-list.scss

```diff
@@ -0,0 +1,28 @@
+@import "../../mixins/typography";
+
+.inline-list {
+  @include list-style-none();
+  font-family: $font-secondary;
```

Comment:
Not convinced the font should be set at all (in case this pattern is used elsewhere).

Reply:
The current use case requires `font-family` to be set. Making the `font-family` configurable seems like complexity that we don't need at the moment. We can revisit if / when we have a need for that flexibility.

Reply:
It's no configuration if it's not set (like we don't fix the font size, colour etc). It does mean we'd have to set the font on the Content Header, but that feels like the actual use case to me.

Reply:
👍 Happy to set it on the content header.